Content/Library/src/MyLib.1/MyLib.1.fsproj

```diff
@@ -2,6 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
Do you think this change makes sense to be in the one of the `Directory.Build.props`? Either root or the one under `src`.

Reply:
Yes, sure. For all projects under src AssemblyInfo.fs files is generated, it is logical that the GenerateAssemblyInfo flag should be set to false in the Directory.Build.props at the src level. About the root level in the next comment.